.vscode/settings.json

```diff
@@ -1,13 +1,7 @@
-// Place your settings in this file to overwrite default and user settings.
 {
-    "files.exclude": {
-        // "badges": true,
-        // "node_modules": true,
-    },
```

Comment:
This must be keeped, I commented to check some things but these folders aren't important to development so they can be hidden